lib/modules/commentQuickCollapse.js

```diff
@@ -11,14 +16,78 @@ module.category = 'commentsCategory';
 module.description = 'commentQuickCollapseDesc';
 
 module.options = {
+	hideCommentsOnHeaderDoubleClick: {
+		type: 'boolean',
+		value: true,
+		description: 'hideCommentsOnHeaderDoubleClickDesc',
+		title: 'hideCommentsOnHeaderDoubleClickTitle',
+	},
+	toggleCommentsOnClickLeftEdge: {
+		type: 'boolean',
+		value: false,
+		bodyClass: true,
+		description: 'toggleCommentsOnClickLeftEdgeDesc',
+		title: 'toggleCommentsOnClickLeftEdgeTitle',
+	},
+	leftEdgeColor: {
+		dependsOn: 'toggleCommentsOnClickLeftEdge',
+		type: 'color',
+		value: '#D1D1D1',
+		description: 'toggleCommentsLeftEdgeColorDesc',
+		title: 'toggleCommentsLeftEdgeColorTitle',
+	},
+	leftEdgeHoverColor: {
+		dependsOn: 'toggleCommentsOnClickLeftEdge',
+		type: 'color',
+		value: '#B1B1B1',
+		description: 'toggleCommentsLeftEdgeHoverColorDesc',
+		title: 'toggleCommentsLeftEdgeHoverColorTitle',
+	},
+	leftEdgeCollapsedColor: {
+		dependsOn: 'toggleCommentsOnClickLeftEdge',
+		type: 'color',
+		value: '#B1B1B1',
+		description: 'toggleCommentsLeftEdgeCollapsedColorDesc',
+	},
+	leftEdgeWidth: {
+		dependsOn: 'toggleCommentsOnClickLeftEdge',
+		type: 'text',
+		value: '100',
+		description: 'toggleCommentsLeftEdgeWidthDesc',
+	},
+	hideCollapseButton: {
+		dependsOn: 'toggleCommentsOnClickLeftEdge',
+		type: 'boolean',
+		value: true,
+		bodyClass: true,
+		description: 'toggleCommentsLeftEdgeHideButtonDesc',
+	},
+	scrollOnCollapse: {
+		dependsOn: 'toggleCommentsOnClickLeftEdge',
```

Comment:
If this is intended to be dependent on `toggleCommentsOnClickLeftEdge`, it should be moved inside the `if (module.options.toggleCommentsOnClickLeftEdge.value) {` block in `module.go`, so it doesn't take effect when the former is disabled.